src/components/modals/export-image-modal.js

```diff
@@ -64,31 +60,34 @@ const ExportImageModalFactory = () => {
       mapH: PropTypes.number.isRequired,
       exportImage: PropTypes.object.isRequired,
       // callbacks
-      onUpdateSetting: PropTypes.func.isRequired
+      onUpdateImageSetting: PropTypes.func.isRequired
     };
 
     componentDidMount() {
-      this._updateMapDim();
+      // this._updateMapDim();
```

Comment:
why is this comment out?

Reply:
that's correct, we no longer need it since _updateMapDim is also called on componentDidUpdate